pandas/core/indexes/datetimes.py

```diff
@@ -1707,8 +1707,69 @@ def freq(self, value):
     weekofyear = _field_accessor('weekofyear', 'woy',
                                  "The week ordinal of the year")
     week = weekofyear
-    dayofweek = _field_accessor('dayofweek', 'dow',
-                                "The day of the week with Monday=0, Sunday=6")
+    _dayofweek_doc = """
+    The day of the week with Monday=0, Sunday=6.
+
+    Return the day of the week. It is assumed the week starts on
+    Monday, which is denoted by 0 and ends on Sunday which is denoted
+    by 6. This method is available on both Series with datetime
+    values or DatetimeIndex
+
+    See Also
+    --------
+    pandas.Series.dt.dayofweek : Identical method.
```

Comment:
you can say alias

Reply:
add day_name as well

Reply:
I've changed it into this now.

```
    See Also
    --------
    pandas.Series.dt.dayofweek : alias
    pandas.Series.dt.weekday : alias
    pandas.Series.dt.day_name : 
        Return the day names of the DateTimeIndex with specified locale.
```

Note that the docstring for `dayofweek` and `weekday` is the exact same. 

```
    dayofweek = _field_accessor('dayofweek', 'dow', _dayofweek_doc)
    weekday = dayofweek
```